src/main/java/Application/Utilities/DetectMarker.java

```diff
@@ -0,0 +1,46 @@
+package Application.Utilities;
+
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.aruco.*;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class DetectMarker {
+   private List<Mat> corners;
+   private Mat ids;
+
+   public DetectMarker() {
+      this.corners = new ArrayList<>();
+      this.ids = new Mat();
+   }
+
+   public void detect(Mat subject) {
+      Dictionary dictionary = Aruco.getPredefinedDictionary(Aruco.DICT_6X6_250);
```

Comment:
Why fixed value?

Reply:
up until this point I only tested with that particular dictionary
but there are not particular reason so it can be taken out as a class variable or loaded from configuration (one exevution uses one library)
also is detection of markers in the same image but from multiple dictionaries within the scope of this application (one execution multiple library)?

Reply:
just add a TODO here